src/OrchardCore.Modules/OrchardCore.OpenId/Controllers/UserInfoController.cs

```diff
@@ -120,8 +120,7 @@ public async Task<IActionResult> Me()
                     }
                     else
                     {
-                        _logger.LogWarning($"{OpenIdConnectConstants.Claims.UpdatedAt} claim value '{updatedAtClaimValue}' is invalid. Should be the time the end-user's" +
-                                           " information was last updated, as number of seconds since the Unix epoch (1970-01-01T0:0:0Z) as measured in UTC until the date/time. ");
+                        _logger.LogWarning("{UpdatedAt} claim value '{UpdatedAtClaimValue}' is invalid. Should be the time the end-user's information was last updated, as number of seconds since the Unix epoch (1970-01-01T0:0:0Z) as measured in UTC until the date/time. ", OpenIdConectConstants.Claims.UpdatedAt, updatedAtClaimvalue);
```

Comment:
> _logger.LogWarning("{0} claim value '{1}' is invalid. Should be the time the end-user's information was last updated, as number of seconds since the Unix epoch (1970-01-01T0:0:0Z) as measured in UTC until the date/time.", OpenIdConnectConstants.Claims.UpdatedAt, updatedAtClaimValue);

I was about to do the change but I don't get it why is has to be changed.
I understand it for strings that have to be translated but why for this kind of log?

Reply:
Personally, I'd prefer if we had an exception instead of a log message. Or we could just use `long.Parse` like we do for booleans (BTW, we should use an overload accepting a `CultureInfo`).

Reply:
Changed to an exception rather than a log warning @kevinchalet 

@agriffard for all logs, shouldn't use the string interpolation syntax of `$"{some-format}"`